manual/conf.py

```diff
@@ -3,7 +3,7 @@
 # Configuration file for the Sphinx documentation builder.
 # See https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-copyright = "2020-2023 Quantinuum"
+COPYRIGHT = "2020-2023 Quantinuum"
```

Comment:
The copyright format is as follows, no?

> A copyright statement in the style '2008, Author Name'.

```suggestion
copyright = "2020-2023, Quantinuum"
```

Reply:
VSCode complains about `copyright` being lowercase. 

I think you're right about it being lowercase, will just ignore VSCode. The comma also should be there.